x/mint/abci.go

```diff
@@ -35,8 +35,11 @@ func BeginBlocker(ctx sdk.Context, k Keeper) {
 	}
 
 	communityPoolRatio := k.GetCommunityPoolRatio(ctx)
-	communityPoolCoins := k.GetCommunityPoolMint(ctx, communityPoolRatio, mintedCoin)
-	collectedFeesCoins := mintedCoins.Sub(communityPoolCoins)
+	communityPoolCoins := k.GetPoolMint(ctx, communityPoolRatio, mintedCoin)
+
+	shieldStakingPurchasePoolRatio := k.GetShieldStakingPurchasePoolRatio(ctx)
+	SPPCoins := k.GetPoolMint(ctx, shieldStakingPurchasePoolRatio, mintedCoin)
+	collectedFeesCoins := mintedCoins.Sub(communityPoolCoins).Sub(SPPCoins)
```

Comment:
So pools do get fees' cut? I thought you said they only get (nominal) inflation? This is fine but just want to confirm this is the case.

Reply:
No, these distribute only the newly minted coins. `collectedFeesCoins` is just named so it adds to the `fee_collector` account which includes collected fees + block provision

Reply:
I meant these should not be hard-wired in the long run.

Reply:
OK SGTM.